connector.py

```diff
@@ -0,0 +1,53 @@
+"""
+Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+
+def get_ephemeral(service, project, instance, pub_key):
+    """
+    A helper function that requests an ephemeral certificate from the
+    Cloud SQL Instance.
+
+    Takes in a service object, the project name, the instance name and
+    a public key.
+
+    Returns the certificate as a string.
+    """
+
+    if (
+        service is None or
+        project is None or
+        instance is None or
+        pub_key is None
+    ):
+        print("Cannot take None as an argument.")
+        exit()
+
+    # TODO(ryachen@) Add checks to ensure service object is valid.
+
+    request = service.sslCerts().createEphemeral(
+        project=project,
+        instance=instance,
+        body={
+            'public_key': pub_key
+        }
+    )
+    response = request.execute()
```

Comment:
If this is a `requests` object you can do `.raise_for_status()` and an exception will be raised if there was an error. https://2.python-requests.org/en/master/api/#requests.Response.raise_for_status

Reply:
Looks like it isn't a `requests` object, but `request.execute()` raises an error when there's a 400 status code. So I guess the check isn't necessary haha.